media-gfx/flameshot/flameshot-0.9.0.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop cmake xdg-utils
+
+DESCRIPTION="Powerful yet simple to use screenshot software"
+HOMEPAGE="https://flameshot.org https://github.com/flameshot-org/flameshot"
+SRC_URI="https://github.com/flameshot-org/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="FreeArt GPL-3 Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64 x86"
```

Comment:
Please reset the KEYWORDS with `ekeyword ~all your.ebuild` whenever doing version bumps.